src/MOI_io.jl

```diff
@@ -50,8 +50,8 @@ end
 
 # get (i,j) matrix indices (lower triangle, row major) for kth term in vector
 function vectomatidx(k)
-    i = isqrt(2*k)
-    j = k - div((i-1)*i, 2)
+    i = floor(Int, (1+sqrt(8*k-7))/2)
```

Comment:
I would feel safer with `div(1 + isqrt(8k-7), 2)`. With the `floor` technique there is always a risk that you get something like `floor(Int, 7.999998) = 7`.

Reply:
```julia
julia> f(k) = floor(Int, (1+sqrt(8*k-7))/2)
f (generic function with 1 method)

julia> g(k) = div(1 + isqrt(8k-7), 2)
g (generic function with 1 method)

julia> all(k -> g(k) == f(k), 1:1000)
true

julia> all(k -> g(k) == f(k), 1:1000000)
true
```

Reply:
Thanks! Great